src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -95,6 +116,30 @@ static int uct_cuda_copy_md_is_dmabuf_supported()
     return dmabuf_supported;
 }
 
+static int uct_cuda_copy_md_is_fabric_supported()
```

Comment:
IMO this function and the check during MD initialization are redundant. If fabric memory is requested and not supported, we would fail (or fallback) anyway during the allocation.

Reply:
removed the check and function